src/fonts.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: "Ubuntu";
+  src: url("../static/fonts/Ubuntu.ttf") format("truetype");
+}
```

Comment:
could use these https://github.com/TradeTrust/tradetrust-ui-components#fonts-management

Reply:
oh nice ok will use those instead

Reply:
ubuntu font is google font, think can load in `<head />` directly instead

Reply:
careful not to accidentally define font entirely for all templates tho. define as a utility tailwind css class maybe, then use as and when needed in places